addons/mass_mailing_crm/__init__.py

```diff
@@ -2,3 +2,19 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from . import models
+
+from odoo import api, SUPERUSER_ID
+
+
+def uninstall_hook(cr, registry):
+    # restore action to send simple mails in mass mode
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    env.ref('crm.action_lead_mass_mail').write(
```

Comment:
Same as other: don't crash if it does not exist, and also update name to its initial value